network-api/networkapi/settings.py

```diff
@@ -233,6 +235,23 @@
     },
 ]
 
+
+#specifies the full Python path to each panel that you want included in the toolbar
+DEBUG_TOOLBAR_PANELS = [
```

Comment:
the toolbar has a default set of panels, so we can probably remove this section entirely.